command/health/cmd/root.go

```diff
@@ -0,0 +1,185 @@
+/********************************************************************************
+ *  Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/oliveagle/jsonpath"
+	"github.com/spf13/cobra"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+)
+
+const ServiceName = "service:"
```

Comment:
I would call this constant `ServicePrefix` instead of `ServiceName` to avoid confusing on the context of serviceName.

Reply:
FYI that's the wrong check anyway.  The keys need to be ignored and the logic needs to match on the ServiceName property.